<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoiEditorControlPanel</class>
 <widget class="QMainWindow" name="RoiEditorControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>613</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>613</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>423</width>
    <height>613</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="main_layout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="margin" stdset="0">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="paths and settings">
      <property name="title">
       <string>Choose paths and settings</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="file_org_layout">
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="label_original">
           <property name="text">
            <string>Original:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="txt_original">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>This is the photograph set as background</string>
           </property>
           <property name="text">
            <string>&lt;no name&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QPushButton" name="btn_browse_original">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear_original">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Clears this filename</string>
           </property>
           <property name="text">
            <string>üóëÔ∏è</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="file_label_layout">
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="label_label">
           <property name="text">
            <string>Label:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="txt_label">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>This is the label file generated by Cellpose</string>
           </property>
           <property name="text">
            <string>&lt;no name&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QPushButton" name="btn_browse_label">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear_label">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Clears this filename</string>
           </property>
           <property name="text">
            <string>üóëÔ∏è</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="file_zip_layout">
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="label_zip">
           <property name="text">
            <string>Zip:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="txt_zip">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>This is a zip generated by the RoiEditor or Cellpose</string>
           </property>
           <property name="text">
            <string>&lt;no name&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QPushButton" name="btn_browse_zip">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear_zip">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Clears this filename</string>
           </property>
           <property name="text">
            <string>üóëÔ∏è</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="process_and_unit_groupboxes">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="proces_labels_unit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>Process labels</string>
           </property>
           <layout class="QVBoxLayout" name="paths_checkbox_layout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QCheckBox" name="cbEdge">
              <property name="toolTip">
               <string>When checked, ROIs at the edge of the image are removed when reading a label file</string>
              </property>
              <property name="text">
               <string>Remove at edge</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbSmall">
              <property name="toolTip">
               <string>When checked ROIs smaller than Min size threshold are removed when reading a label file</string>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="text">
               <string>Remove small</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="min_size_layout">
              <item>
               <widget class="QLabel" name="label_thresh">
                <property name="toolTip">
                 <string>When the corresponding checkbox is checked, ROIS smaller than this value will be deleted</string>
                </property>
                <property name="text">
                 <string>Min area:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="tbSize">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string/>
                </property>
                <property name="text">
                 <string>100</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="process_labels_unit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>225</width>
             <height>125</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>225</width>
             <height>125</height>
            </size>
           </property>
           <property name="title">
            <string>Unit &amp;&amp; Scale</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>30</y>
              <width>204</width>
              <height>87</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QRadioButton" name="rb_unit_from_file">
               <property name="minimumSize">
                <size>
                 <width>77</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>77</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>When checked, the microns/pixel are used as read from the image</string>
               </property>
               <property name="text">
                <string>from file:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="le_scale_from_file">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>65</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
                    border: none;
                }
                </string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_unit_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>39</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>39</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Œºm/px</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="rb_unit_specified">
               <property name="minimumSize">
                <size>
                 <width>77</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>77</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>When checked, the microns/pixel as specified here will be used</string>
               </property>
               <property name="text">
                <string>specified:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_custom_scale">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>65</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maxLength">
                <number>12</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_unit_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>39</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>39</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Œºm/px</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="rb_pixel_is_unit">
               <property name="toolTip">
                <string>When checked, the unit of size is set to 1 pixel, ignoring data from the image</string>
               </property>
               <property name="text">
                <string>pixel is unit</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_next">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Opens the files and shows the image</string>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Edit ROIs">
      <property name="title">
       <string>Edit ROIs</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="top_button_layout">
         <item>
          <widget class="QPushButton" name="btn_outer">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Selects the ROIs on the outer edge</string>
           </property>
           <property name="text">
            <string>Select Outer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_saveRois">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save the ROIS to a zip file</string>
           </property>
           <property name="text">
            <string>Save ROIs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="middle_button_layout">
         <item>
          <widget class="QPushButton" name="btn_outliers">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selects all ROIs for the current measurement outside  of med+/-1.5*IQR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Select Outliers</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_saveTable">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Save Measurements</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="edit_checkbox_layout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QGroupBox" name="Show">
           <property name="maximumSize">
            <size>
             <width>135</width>
             <height>128</height>
            </size>
           </property>
           <property name="title">
            <string>Show</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QCheckBox" name="cb_show_names">
              <property name="text">
               <string>Show names</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_show_deleted">
              <property name="text">
               <string>Show deleted</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_show_overlay">
              <property name="text">
               <string>Show overlay</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="prev_and_finish">
         <item>
          <widget class="QPushButton" name="btn_prev">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Watch out! This button will close the image window</string>
           </property>
           <property name="text">
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_finish">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Watch out! This button closes the application</string>
           </property>
           <property name="text">
            <string>Finish</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
